lib/Yam/Agama/Pom/AgamaUpAndRunningPage.pm

```diff
@@ -16,7 +16,7 @@ use testapi;
 sub new {
     my ($class, $args) = @_;
     return bless {
-        tag_array_ref_any_first_screen_shown => [qw(agama-product-selection agama-configuring-the-product agama-installing)]
+        tag_array_ref_any_first_screen_shown => [qw(agama-product-selection agama-configuring-the-product agama-installing agama-sle-overview)]
```

Comment:
never pick big areas for matching, you can pick an area for the text "SUSE Linux Enterprise Server" and another area for text "Overview":  https://openqa.suse.de/tests/15527936#step/boot_agama/3

Reply:
5b383abf040e62338fbdcf8380f91a2efe572ec5